types/utm/index.d.ts

```diff
@@ -19,7 +20,7 @@ export function toLatLon(
     northing: number,
     zoneNum: number,
     zoneLetter: string,
-    northern: boolean,
+    northern?: boolean,
```

Comment:
The js code seems to indicate that exactly one of either `zoneLetter` or `northern` should be set. So this should probably be an overload:
```
export function toLatLon(<args>, zoneLetter: string, northern?: undefined, strict?: boolean);
export function toLatLon(<args>, zoneLetter: undefined <| any other falsy values>, northern: boolean, strict?: boolean);
```